build/templates/tox-system_tests.ini.mako

```diff
@@ -57,19 +61,36 @@ commands =
     # token is from codecov
     ${module_name}-coverage: codecov -X gcov --token=4c58f03d-b74c-489a-889a-ab0a77b7809f --no-color --flags ${module_name}systemtests --name ${module_name} --root ../.. --file ../../generated/${module_name}/coverage.xml
 
+    ${module_name}-test_suite: python --version
```

Comment:
I think `print_diagnostic_information` includes version. It may also include platform.architecture.
If it doesn't, then we should consider making it so it does so se can simplify these files and also have more control over formatting.

Reply:
Removed printing the version and architecture separately.